csrc/python_frontend/python_bindings.cpp

```diff
@@ -2921,6 +2942,65 @@ void initNvFuserPythonBindings(PyObject* module) {
       py::arg("factor"),
       py::arg("inner_split") = true,
       py::arg("trim_out_of_bounds") = false);
+  nvf_sched.def(
+      "cache_after",
+      [](FusionDefinition::SchedOperators& self,
+         Tensor tensor,
+         const LoadStoreOpType& op_type,
+         const CacheOp& cache_op) -> Tensor {
+        NVF_CHECK(
+            self.validUse(),
+            "Attempting to use a SchedOperators Op prior to definition!");
+        FusionDefinition* fd = self.fusion_definition;
+        TensorView* input_tv =
+            fd->getFusionState(tensor.index)->template as<TensorView>();
+        TensorView* output_tv = input_tv->cacheAfter(op_type, cache_op);
+        Tensor output = fd->defineTensor(tensor.dims);
+        NVF_CHECK(
+            output.index == fd->numFusionStates(),
+            "Fusion State index does not match the size!");
+        fd->addFusionState(output_tv);
```

Comment:
this pattern is showing up enough times I think we can add a function in FusionDefinition as `Tensor FusionDefinition::addTensor(TensorView* tv)`.

I think `tensor.dims == output_tv->nDims()`.

Reply:
Giving that this is the only use case for addFusionState, maybe we can change that one instead.